src/util/simplify_expr.cpp

```diff
@@ -172,6 +172,28 @@ simplify_exprt::simplify_clz(const count_leading_zeros_exprt &expr)
   return from_integer(n_leading_zeros, expr.type());
 }
 
+simplify_exprt::resultt<>
+simplify_exprt::simplify_ctz(const count_trailing_zeros_exprt &expr)
+{
+  const auto const_bits_opt = expr2bits(
```

Comment:
Does it make sense to use the lowering in the simplifier to avoid the duplicate implementation?

Reply:
We could, but it's probably more expensive. Whether that's a practical concern, however, is a different question. I suppose ctz will be fairly rare.